shared/tracker/render.desktop.js

```diff
@@ -20,11 +32,40 @@ export default class Render extends Component {
     // So we use the existing paddingBottom and add the height of the footer
     const footerHeight = calcFooterHeight(this.props.actionProps.loggedIn)
     const calculatedPadding = styles.content.paddingBottom + footerHeight
+
+    let backgroundStyle = (this.props.headerProps.currentlyFollowing && !this.props.headerProps.changed) ? styles.headerSuccess : styles.headerNormal
```

Comment:
really this needs to go up a level and be more generic so it could be shared w/ mobile. this code would then be like
```
switch (this.props.trackerState) {
case 'warning': backgroundStyle = {backgroundColor: yellow}
```
or something, but i didn't want to change that much code yet